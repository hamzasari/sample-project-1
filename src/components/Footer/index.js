import React from 'react';
import { Translate } from 'react-redux-i18n';

/**
 * Footer component
 * This component is shown at the bottom of all pages
 * @returns {JSX.Element}
 */
const Footer = () => {
  return (
    <footer className="footer">
      <div className="content has-text-centered">
        <p>
          <strong>
            <Translate value={`headerLinks.title`} />
          </strong>
          <br />
          Hamza SARI
          <br />
          2021
        </p>
      </div>
    </footer>
  );
};

export default Footer;
