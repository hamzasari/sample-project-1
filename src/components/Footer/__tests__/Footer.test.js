import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../index';

it('renders Footer snapshot', () => {
  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
});
