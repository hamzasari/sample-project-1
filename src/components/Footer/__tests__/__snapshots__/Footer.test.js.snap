// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Footer snapshot 1`] = `
<footer
  className="footer"
>
  <div
    className="content has-text-centered"
  >
    <p>
      <strong>
        <span>
          title
        </span>
      </strong>
      <br />
      Hamza SARI
      <br />
      2021
    </p>
  </div>
</footer>
`;
